internal/runners/initialize/init.go

```diff
@@ -161,23 +186,12 @@ func (r *Initialize) Run(params *RunParams) (rerr error) {
 		}
 	}
 
-	// Match the case of the organization.
-	// Otherwise the incorrect case will be written to the project file.
-	var owner string
-	orgs, err := model.FetchOrganizations()
+	resolvedOwner, err = r.getOwner(paramOwner)
 	if err != nil {
-		return errs.Wrap(err, "Unable to get the user's writable orgs")
-	}
-	for _, org := range orgs {
-		if strings.EqualFold(org.URLname, params.Namespace.Owner) {
-			owner = org.URLname
-			break
-		}
+		return locale.WrapError(err, "err_init_owner", "Could not determine owner")
```

Comment:
Please use `errs.Wrap()`. The rationalizer should be handling this and reporting a user-facing error, and I think it does for `errNoOwner` at least.